common/djangoapps/util/enterprise_helpers.py

```diff
@@ -120,3 +124,62 @@ def insert_enterprise_pipeline_elements(pipeline):
 
     for index, element in enumerate(additional_elements):
         pipeline.insert(insert_point + index, element)
+
+
+def get_enterprise_customer_logo_url(request):
```

Comment:
As we are utilizing the request context to store filter parameters here and to keep context simple I think, We should keep this method here.